docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -567,29 +689,21 @@ $ terraform apply
 In the terminal, Terraform displays a message that `rhoas_service_account.my-service-account` will be created.
 . When you're ready to create your service account, type *yes*. The generated client ID appears in the terminal as an output. The client secret does not appear because it is marked as a sensitive value.
 . To verify Terraform successfully created your service account, in your web browser, open the *Service Accounts* page of the {product-kafka} {service-url-kafka}[web console^].
-. To create a Kafka topic with default values, copy and paste the `rhoas_topic` resource shown in the following example into the `main.tf` file. This example uses the `topic` identifier and creates the `my-topic` Kafka topic. As you have already created the Kafka instance, Terraform can check dependencies for this new topic resource and knows the Kafka ID when you run this example resource.
-+
-.Example `rhoas_topic` resource with default values
-[source]
-----
-resource "rhoas_topic" "topic" {
-		name = "my-topic"
-		partitions = 1
-		kafka_id = rhoas_kafka.instance.id
-	}
 
-----
-+
-. Save your changes.
-. Apply the changes you made to your Terraform provider configuration.
-+
-[source, shell]
-----
-$ terraform apply
-----
-In the terminal, Terraform displays a message that `rhoas_topic.my-topic` will be created.
-. When you're ready to create your topic, type *yes*.
-. To verify Terraform successfully created your topic, in your web browser, open the *Topics* page of the {product-kafka} {service-url-kafka}[web console^].
+[id="proc-creating-acl-permissions-terraform_{context}"]
+==== Creating Access Control List permissions
+
+[role="_abstract"]
+After you create a service account to connect to a Kafka instance, you must also set the appropriate level of access for that new account in an Access Control List (ACL) for the Kafka instance.
+
+.Prerequisites
+
+* You have an offline token that authenticates the Terraform resources with the {product-long-rhoas} API.
+* You have created a Kafka instance.
+* You understand how Access Control Lists (ACLs) enable you to manage how user accounts and service accounts can access the Kafka resources that you create. See {base-url}{access-mgmt-url-kafka}[Managing account access in {product-long-kafka}^].
+
+.Procedure
+
 . To create Access Control List (ACL) permissions with some default values, copy and paste the `rhoas_acl` resource shown in the following example into the `main.tf` file. This example uses the `"acl"` identifier.
```

Comment:
Think you probably want to show the code block as part of step 1, and not at the bottom of step 2.